### ISSUE TITLE
Expandng buttons now animate

### ISSUE THREAD

**Issue by Andre-Pham** [Andre-Pham/yonder#6]
Xcode 14.0.0, iOS 16.0.0 simulator.
`YonderExpandableWideButtonBody` now expands with an unintended animation. This will need to be reverted back to its original animation of immediately expanding.

**Comment by Andre-Pham** [comment]
This can be resolved by changing `animation` in `OptionsStateManager.swift` to be static and making the following changes:

```
// YonderWideButtonBody.swift
// ...
private let permittedAnimations = [OptionsStateManager.animation]
// ...
content()
    // ...
    .transaction { transaction in
        if !self.permittedAnimations.contains(where: { $0 == transaction.animation }) {
            transaction.animation = nil
        }
    }
```

I won't make the changes until Xcode 14 and iOS 16 are more mature, as this may be an unintended change on Apple's part.

**Comment by Andre-Pham** [comment]
Xcode 14.2.0, iOS 16.2 simulator, plus tested on a real device.
It seems this is an intentional change on Apple's part unless they're really taking their time reverting the changes.
Resolved in ea43f852206ba1cc13f28bea9c0eb3607b066b3d.
